Failing test `BaseHiveConnectorTest.testTargetMaxFileSize`

The test `io.trino.plugin.hive.TestHiveConnectorTest.testTargetMaxFileSize` is sometimes failing

https://github.com/trinodb/trino/actions/runs/4340988870/jobs/7580223072


```
Error:  io.trino.plugin.hive.TestHiveConnectorTest.testTargetMaxFileSize  Time elapsed: 1.79 s  <<< FAILURE!
io.trino.testing.QueryFailedException: slideWindowSize must be positive
	at io.trino.testing.AbstractTestingTrinoClient.execute(AbstractTestingTrinoClient.java:122)
	at io.trino.testing.DistributedQueryRunner.execute(DistributedQueryRunner.java:489)
	at io.trino.testing.QueryAssertions.assertUpdate(QueryAssertions.java:71)
	at io.trino.testing.AbstractTestQueryFramework.assertUpdate(AbstractTestQueryFramework.java:411)
	at io.trino.plugin.hive.BaseHiveConnectorTest.testTargetMaxFileSize(BaseHiveConnectorTest.java:1899)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: java.lang.Exception: SQL: CREATE TABLE test_max_file_size WITH (format = 'TEXTFILE') AS SELECT * FROM tpch.sf1.lineitem LIMIT 1000000
		at io.trino.testing.DistributedQueryRunner.execute(DistributedQueryRunner.java:492)
		... 16 more
Caused by: io.trino.spi.TrinoException: slideWindowSize must be positive
	at io.trino.plugin.hive.line.LineFileWriter.appendRows(LineFileWriter.java:104)
	at io.trino.plugin.hive.HiveWriter.append(HiveWriter.java:85)
	at io.trino.plugin.hive.HivePageSink.writePage(HivePageSink.java:346)
	at io.trino.plugin.hive.HivePageSink.doAppend(HivePageSink.java:298)
	at io.trino.plugin.hive.HivePageSink.lambda$appendPage$3(HivePageSink.java:284)
	at io.trino.hdfs.authentication.HdfsAuthentication.lambda$doAs$0(HdfsAuthentication.java:26)
	at io.trino.hdfs.authentication.NoHdfsAuthentication.doAs(NoHdfsAuthentication.java:25)
	at io.trino.hdfs.authentication.HdfsAuthentication.doAs(HdfsAuthentication.java:25)
	at io.trino.hdfs.HdfsEnvironment.doAs(HdfsEnvironment.java:98)
	at io.trino.plugin.hive.HivePageSink.appendPage(HivePageSink.java:284)
	at io.trino.plugin.base.classloader.ClassLoaderSafeConnectorPageSink.appendPage(ClassLoaderSafeConnectorPageSink.java:69)
	at io.trino.operator.TableWriterOperator.addInput(TableWriterOperator.java:258)
	at io.trino.operator.Driver.processInternal(Driver.java:399)
	at io.trino.operator.Driver.lambda$process$8(Driver.java:297)
	at io.trino.operator.Driver.tryWithLock(Driver.java:689)
	at io.trino.operator.Driver.process(Driver.java:289)
	at io.trino.operator.Driver.processForDuration(Driver.java:260)
	at io.trino.execution.SqlTaskExecution$DriverSplitRunner.processFor(SqlTaskExecution.java:773)
	at io.trino.execution.executor.PrioritizedSplitRunner.process(PrioritizedSplitRunner.java:165)
	at io.trino.execution.executor.TaskExecutor$TaskRunner.run(TaskExecutor.java:523)
	at io.trino.$gen.Trino_testversion____20230306_130054_11118.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalArgumentException: slideWindowSize must be positive
	at io.airlift.compress.zstd.Util.checkArgument(Util.java:52)
	at io.airlift.compress.zstd.CompressionContext.slideWindow(CompressionContext.java:47)
	at io.airlift.compress.zstd.ZstdOutputStream.writeChunk(ZstdOutputStream.java:209)
	at io.airlift.compress.zstd.ZstdOutputStream.compressIfNecessary(ZstdOutputStream.java:126)
	at io.airlift.compress.zstd.ZstdOutputStream.write(ZstdOutputStream.java:102)
	at io.airlift.compress.zstd.ZstdHadoopOutputStream.write(ZstdHadoopOutputStream.java:48)
	at io.airlift.slice.Slice.getBytes(Slice.java:648)
	at io.trino.hive.formats.line.text.TextLineWriter.write(TextLineWriter.java:64)
	at io.trino.plugin.hive.line.LineFileWriter.appendRows(LineFileWriter.java:97)
	... 23 more

```

--- a/pom.xml
+++ b/pom.xml
@@ -820,7 +820,7 @@
             <dependency>
                 <groupId>io.airlift</groupId>
                 <artifactId>aircompressor</artifactId>
-                <version>0.22</version>
+                <version>0.24</version>
             </dependency>
 
             <dependency>

